src/encode.test.ts

```diff
@@ -13,148 +13,31 @@ import {
 	removeDiacritics,
 	serialize,
 } from "./encode.js";
+import {
+	payloadWithDirectDebit,
+	payloadWithPaymentOrder,
+	payloadWithStandingOrder,
+	serializedDirectDebit,
+	serializedPaymentOrder,
+	serializedStandingOrder,
+} from "./test_assets.js";
```

Comment:
I have extracted some variables from the `encode.test.ts` file and moved them to the `test_assets.ts` file so they can be reused in `decode.test.ts`.

Reply:
I expect this would happen as the tests increase :+1:, but I'm still not decided on how to name the data variables. Should one file hold all the test data, and what simple name should we choose for clarity?

Reply:
For now, I store only shared variables in `test_assets.ts`. The file name and names of variables can be changed, do you have specific names in mind?

Reply:
Not really, the naming you chose is good